File: CS/MyBandedGridView/MyBandedGridView.cs
using System;

namespace CustomGrid {
	public class MyBandedGridView : DevExpress.XtraGrid.Views.BandedGrid.BandedGridView {
		public MyBandedGridView() : this(null) {}
		public MyBandedGridView(DevExpress.XtraGrid.GridControl grid) : base(grid) {}
		protected override string ViewName { get { return "MyBandedGridView"; } }
	}
}

File: VB/MyBandedGridView/MyBandedGridView.vb
Namespace CustomGrid

    Public Class MyBandedGridView
        Inherits DevExpress.XtraGrid.Views.BandedGrid.BandedGridView

        Public Sub New()
            Me.New(Nothing)
        End Sub

        Public Sub New(ByVal grid As DevExpress.XtraGrid.GridControl)
            MyBase.New(grid)
        End Sub

        Protected Overrides ReadOnly Property ViewName As String
            Get
                Return "MyBandedGridView"
            End Get
        End Property
    End Class
End Namespace
